docs/weaviate.md

```diff
@@ -0,0 +1,18 @@
+# Weaviate-specific Setup
+
+1. Deploy weaviate using docker compose
+   Here's a bare minimum docker compose file:
+   ```bash
+   curl -o docker-compose.yml "https://configuration.weaviate.io/v2/docker-compose/docker-compose.yml?modules=standalone&runtime=docker-compose&weaviate_version=v1.19.3"
+   ```
+
+2. Install Langchain's PR #4365:
```

Comment:
I feel we should wait until this is part of LangChain main branch before merging this, in order to not increase the complexity of the setup.

Reply:
👍